draft-ietf-ppm-dap.md

```diff
@@ -2029,21 +2021,19 @@ variant {
 If `state == Continued(prep_state)`, then the Helper stores `state` to
 prepare for the next continuation step ({{aggregation-helper-continuation}}).
 
-If `state == Finished(out_share)`, the Helper MUST resolve replay of the
-report. It does so by checking if the report ID was previously aggregated for
-this task. If so, it responds with
+If `state == Finished(out_share)`, then the Helper commits to `out_share` as
+described in {{batch-buckets}}. If commitment fails with some `ReportError`
+value `commit_error`, then instead of the `continue` response above, the Helper
```

Comment:
I now see what you're doing. But "`commit_error`" reads as if `commit_error` is a variant of `ReportError`. I would just do this:
```suggestion
, then instead of the `continue` response above, the Helper
```

Reply:
I would also clarify how this might happen in a parantehtical:

> (i.e., the report was replayed or its batch bucket was collected)